src/app/shared/dovetail/dovetail-audio.ts

```diff
@@ -333,52 +312,40 @@ export class DovetailAudio extends ExtendableAudio {
   }
 
   private skipToFile(index: number, resume: false|PromiseResolver<void> = false): boolean {
-    if (this.index !== index && this.arrangement.entries.length > index) {
+    if (this.index !== index && this.arrangement.length > index) {
       if (this.index !== -1) {
-        const eventData: SegmentEventData = {
-          segment: this.arrangement.entries[this.index],
-          segmentType: this.arrangement.entries[this.index].type
-        };
-        this.$$sendEvent(SEGMENT_END, eventData);
+        this.$$sendEvent(SEGMENT_END, {segment: this.arrangement[this.index]});
       }
       this.index = index;
-      let arrangement = this.arrangement.entries[index];
```

Comment:
It was confusing to also call this an arrangement.

Reply:
++